src/Session.cpp

```diff
@@ -191,7 +191,7 @@ bool SESSION::CSession::CheckPlayableStreams(PLAYLIST::CPeriod* period)
                 //! the task to initialize DRM session only to the requested streams.
                 //! It can be tested e.g. with some Am@zon videos
                 auto& caps = session->capabilities;
-
+#ifndef TARGET_WEBOS
```

Comment:
you can move this define into the appropriate `IsSecureDecoderAudioSupported` method
here there are a lot of workaround due to this missing secure audio decoder feature